charts/victoria-metrics-k8s-stack/README.md.gotmpl

```diff
@@ -0,0 +1,138 @@
+# Helm Chart For Victoria Metrics kubernetes monitoring stack.
+
+{{ template "chart.typeBadge" . }} {{ template "chart.versionBadge" . }}
+
+{{ template "chart.description" . }}
+
+# Prerequisites
+
+* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](../../REQUIREMENTS.md).
+
+* PV support on underlying infrastructure.
```

Comment:
Also, suggest to add following requirements:
```bash
 helm repo add grafana https://grafana.github.io/helm-charts
 helm repo add ksm https://kubernetes.github.io/kube-state-metrics 
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts 
 helm update
```

Reply:
added

Reply:
Exists ``helm update`` command for update repo?

Wouldn't the correct ``helm repo update`` be backwards compatible?

Reply:
Thanks, changed it for helm repo update